src/functions-and-arrays.js

```diff
@@ -28,9 +132,40 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
+function uniquifyArray(word){
+
+  let newArrayWithoutDuplicates=[];
+  if(word.length === 0){
+    return null
+  }
+for(let i=0; i<word.length; i++){
+  const currentWord = word[i];
+if(newArrayWithoutDuplicates.indexOf(currentWord) === -1){
+    newArrayWithoutDuplicates.push(currentWord)
+  }
+}
+return newArrayWithoutDuplicates
+}
+
+uniquifyArray(wordsUnique)
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
+function doesWordExist(arr, word){
+if(arr.length === 0){
+  return null
+}
+for(i=0; i < arr.length; i++){
+  var currentWord = arr[i];
+  if (currentWord === word){
+    return true
+  }
+}
+ if(currentWord!= word){
+  return false
+}
+}
```

Comment:
muy bien!
el último if() no hace falta, podemos devolver `false` directamente fuera del for loop (ya que en el caso de que se encontrará la palabra, hemos salido de la función con "return true", y nunca se ejecutará el código debajo)